honeybadgermpc/batch_reconstruction.py

```diff
@@ -134,7 +150,7 @@ def with_at_most_non_none(data, k):
     benchLogger = logging.LoggerAdapter(
         logging.getLogger("benchmark_logger"), {"node_id": myid})
 
-    def point(i): return Fp(i+1)  # TODO: make it use omega
+    point = EvalPoint(Fp, n, use_fft=use_fft)
```

Comment:
nit: Generally a bad practice to use the same variable name as the arg name `use_fft=use_fft`

Reply:
Why?

Reply:
Andrew and I had a discussion the other day where we wanted to actually avoid code like `x = open(x)` and this reminded me of that - that would ease debugging and improve readability but I guess for function args those arguments don't hold.

I guess it is okay to leave it as is.

Reply:
All right.